### ISSUE TITLE
RumtimeError: Expected to have have finished reduction in the prior iteration before starting a new one

### ISSUE THREAD

**Issue by ge35tay** [OpenGVLab/InternVideo#207]
Hi interVideo Team, I try to run the pretrain and use 1B-41 as pretrained model but I meet this following error, how can I solve this?
<img width="891" alt="屏幕截图 2024-11-07 163507" src="https://github.com/user-attachments/assets/a10e794d-3161-4f14-a5ee-ed2b4f453339">

**Comment by ge35tay** [comment]
I try to set find_unused_parameter as true but meet another issue
<img width="874" alt="屏幕截图 2024-11-07 163938" src="https://github.com/user-attachments/assets/5e2f376d-62f0-4fff-9d7a-cb6d5c8a3bc1">

**Comment by ge35tay** [comment]
Besides, in readme it's said that CLIP teacher is necessary in pretrain, but how to set the readme to set this? I only meet the above error when I set clip teacher as InternVL-6B

**Comment by ge35tay** [comment]
<img width="1034" alt="image" src="https://github.com/user-attachments/assets/993c928f-f953-4bb1-90c9-d59e21a58853">
I finally got pretrain running with this config. But if I just want to finetune with VTC and VTM error, what should I do?
